### PR TITLE
sunxi: add testing 5.4 kernel

### PR DESCRIPTION
**Pull request by abajk**
These patches was removed, because they are already in mainline 5.4 kernel:
* 010-v5.3-drivers-ata-ahci_sunxi-Increased-SATA-AHCI-DMA-TX-RX.patch
* 101-arm64-dts-allwinner-a64-Enable-A64-timer-workaround.patch

Compiled and tested on Orange PI One (cortexa7) and Orange Pi PC2 (cortexa53).

### CODE REVIEW

File: target/linux/generic/config-5.4 [hunk 1]
```diff
@@ -170,10 +170,12 @@ CONFIG_ANON_INODES=y
 # CONFIG_AR8216_PHY is not set
```
Comment (ynezz) [inline target/linux/generic/config-5.4]: Changes to the generic config should go into separate commit.

File: target/linux/sunxi/config-5.4 [hunk 2]
```diff
@@ -1,24 +1,32 @@
+CONFIG_64BIT_TIME=y
 # CONFIG_AHCI_SUNXI is not set
 CONFIG_ALIGNMENT_TRAP=y
-# CONFIG_ARCH_AXXIA is not set
```
Comment (ynezz) [inline target/linux/sunxi/config-5.4]: It would be nice to get the refresh of configs into separate commits and due to the number of changes in this file (+48 -53) and a53's I would probably try to make the review easier by splitting it into separate commits per subtarget as well.

File: target/linux/sunxi/patches-5.4/010-v5.3-drivers-ata-ahci_sunxi-Increased-SATA-AHCI-DMA-TX-RX.patch [hunk 3]
```diff
@@ -1,94 +0,0 @@
-From 120357ea176e420d313cf8cf2ff35fbe233d3bab Mon Sep 17 00:00:00 2001
```
Comment (ynezz) [inline target/linux/sunxi/patches-5.4/010-v5.3-drivers-ata-ahci_sunxi-Increased-SATA-AHCI-DMA-TX-RX.patch]: Refresh of patches should go into separate commit as well.

File: target/linux/sunxi/config-5.4 [hunk 4]
```diff
@@ -0,0 +1,605 @@
+# CONFIG_AHCI_SUNXI is not set
```
Comment (ynezz) [inline target/linux/sunxi/config-5.4]: Nitpick, it's obvious, but you're saving the commiter some time if you add missing commit description, otherwise the commiter is going to add one anyway.

**Comment by olek210** [comment]
[WiP]
Commit "kernel: sort generic configuration" moved into separate [PR](https://github.com/openwrt/openwrt/pull/2817).

**Comment by ynezz** [comment]
> Commit "kernel: sort generic configuration" moved into separate PR.

Uh, why? Did I wrote separate PR? I wrote separate *commit* :-)

**Comment by olek210** [comment]
@ynezz Done. Patches are splitted into smaller pieces. Now should be easier to review patches :)
I now, but it can be merged independently and generic config changes very often.

**Comment by ynezz** [comment]
Merged, thanks!
